.github/workflows/linux.yml

```
@@ -222,3 +223,5 @@ jobs:
       run: |
         source /opt/intel/oneapi/setvars.sh
         make -j 2
+    - name: quicktest
+      run: make test
```

```suggestion
    - name: quicktest
      run: |
        source /opt/intel/oneapi/setvars.sh
        make test
```
You need to set the environmental variables again. I *think* this should solve all issues related to missing libraries `libsvml` (Short vector math library) and `libimf` (Intel Math Library)